doc/whats_new.rst

```diff
@@ -414,6 +414,11 @@ API changes summary
      (`#7187 <https://github.com/scikit-learn/scikit-learn/pull/7187>`_)
      by `YenChen Lin`_.
 
+    - ``classes`` parameter was renamed to ``labels`` in
+      :func:`metrics.classification.hamming_loss`.
+      (`#7260 <https://github.com/scikit-learn/scikit-learn/pull/7260>`_) by
+      `Sebastian Vanrell`_.
```

Comment:
you probably want to add a link definition for your name at the bottom of the file